index.js

```diff
@@ -68,7 +68,26 @@ module.exports = function (str) {
   } else if ((match = str.match(new RegExp(regexps['re1'], 'i')))) {
     let days = match[1];
     let period = match[2];
+    if (period === 'year') {
+      let year = new Date().getFullYear() - days;
+      if (year < 0) {
+        return 'Before B.C.';
+      } else {
+        return [true, new Date(new Date().setFullYear(year))];
+      }
+    } else if (period === 'month') {
+      let extraYear = Math.floor(days / 12);
+      let year = new Date().getFullYear() - extraYear;
+      var month = new Date().getMonth();
+      let extraMonth = (month - days) % 12;
+      if (extraMonth >= 0) {
+        month = extraMonth;
+      } else {
+        month = 12 + extraMonth;
+      }
 
+      return [true, new Date(new Date(new Date(new Date().setFullYear(year)).setMonth(month)))];
```

Comment:
Moving away from this format. use this https://github.com/icyflame/relative-date-reverse/commit/68757bc655f21c00a7b166f384dfaa084a252421